src/admin.rs

```diff
@@ -301,6 +302,12 @@ where
             }
             Command::FactoryReset => {
                 debug_now!("Factory resetting the device");
+                let res = syscall!(self.trussed.confirm_user_present(15 * 1000)).result;
+                if let Err(_err) = res {
+                    debug_now!("Failed to verify user presence: {_err:?}");
+                    response.push(FACTORY_RESET_ERROR).ok();
```

Comment:
Why do we need this error code?  I think it does not even do anything because the response is only sent in the `Ok` case.

Reply:
Nevermind, fixed in the next commit.